### PR TITLE
Add Windows 

### PR DESCRIPTION
**Pull request by tin2tin**
Add support for Windows, however ffmpeg must be "installed" using the "Add to path" part here: http://blog.gregzaal.com/how-to-install-ffmpeg-on-windows/ 

This may be too much to ask of the users, and maybe a way to add a path to ffmpeg in the add-on preferences could be considered?

### CODE REVIEW

File: __init__.py [hunk 1]
```diff
@@ -278,6 +303,15 @@ def start_recording(self, context) -> bool:
...
             )
+        elif os_platform == 'Windows':
+            filepath = chr(34)+(self.filepath)+chr(34)#.replace(chr(92), chr(92)+chr(92))+chr(34)
+            ffmpeg_command = (
+                f"ffmpeg -f dshow "
+                #f'-i ":{addon_prefs.audio_device_windows}" '         
+                f'-i audio="Microphone (Realtek(R) Audio)" '
+                f"-framerate {framerate} {filepath}"
+            )
+            print(ffmpeg_command)
```
Comment (fsiddi) [inline __init__.py]: Please do not use print statements in the code. Consider using log.debug instead.

File: __init__.py [hunk 2]
```diff
@@ -278,6 +303,15 @@ def start_recording(self, context) -> bool:
                 f'-i ":{addon_prefs.audio_device_darwin}" '
                 f"-framerate {framerate} {self.filepath}"
             )
+        elif os_platform == 'Windows':
+            filepath = chr(34)+(self.filepath)+chr(34)#.replace(chr(92), chr(92)+chr(92))+chr(34)
```
Comment (fsiddi) [inline __init__.py]: Make sure you run `black` for code formatting.

File: README.md [hunk 3]
```diff
@@ -29,3 +29,6 @@ Alternatively: this repository can be **cloned** and linked to the `scripts/addo
 ### Requirements
 - Blender 2.83 or later.
 - Linux system and `ffmpeg`.
+
+### Installing ffmpeg on Windows
+- Follow the "Add to path" part here(and reboot): http://blog.gregzaal.com/how-to-install-ffmpeg-on-windows/
```
Comment (fsiddi) [inline README.md]:
```suggestion
- Follow the "Add to path" section here (and reboot): http://blog.gregzaal.com/how-to-install-ffmpeg-on-windows/ 
```

**Review by fsiddi — CHANGES_REQUESTED** [review]
A few minor changes are required.

File: __init__.py (hunk 2 above)
Comment (tin2tin) [reply __init__.py]: This and the other comment has been fixed in the updated patch, however the conflict found by github looks absurd to me?

**Review by fsiddi — CHANGES_REQUESTED** [review]
The `black` configuration in this repo has `skip_string_normalization = true`. Please just change the line originally mentioned in the code review, not the entire codebase.

**Comment by tin2tin** [comment]
> The black configuration in this repo has skip_string_normalization = true. Please just change the line originally mentioned in the code review, not the entire codebase.

Clueless about what to do here? How do I use formatting of github? Or I don't understand?

**Comment by tin2tin** [comment]
I hope everything is okay now? 

Btw. i moved the header button to the right side, as it is custom in Blender to have those on the right hand side of the headers.

File: __init__.py [hunk 4]
```diff
@@ -466,9 +500,10 @@ def draw(self, context):
         col.separator()
         col.prop(addon_prefs, "audio_input_device")
         # DEBUG
-        if os_platform == 'Darwin':
-            col.prop(addon_prefs, "audio_device_darwin", text="(Debug)")
-
+#        if os_platform == 'Darwin':
+#            col.prop(addon_prefs, "audio_device_darwin", text="(Debug)")
+#        if os_platform == 'Windows':
+#            col.prop(addon_prefs, "audio_device_windows", text="(Debug)")
```
Comment (fsiddi) [inline __init__.py]: Please indent the comment (# sign) properly.

File: __init__.py [hunk 5]
```diff
@@ -426,6 +458,8 @@ def draw_push_to_talk_button(self, context):
     layout = self.layout
     layout.enabled = os_platform in supported_platforms
 
+    layout.separator_spacer()
```
Comment (fsiddi) [inline __init__.py]: For now please do not add this. Moving the button to the side makes is difficult to interact with.

**Review by fsiddi — CHANGES_REQUESTED** [review]

File: __init__.py (hunk 4 above)
Comment (tin2tin) [reply __init__.py]: Ha, the joys of coding in the Blender Text Editor. Fixed in updated patch.

File: __init__.py (hunk 5 above)
Comment (tin2tin) [reply __init__.py]: Removed in the updated patch.

**Review by fsiddi — APPROVED** [review]

**Pull request merged**